src/mpl/include/mpl_shm_win.h

```diff
@@ -19,7 +19,7 @@ typedef char *MPLI_shm_ghnd_t;
  *  handle.
  */
 typedef struct MPLI_shm_lghnd_t {
-    MPLI_shm_lhnd_t lhnd;
+    MPL_shm_lhnd_t lhnd;
```

Comment:
Should we fix line 13 instead? The other shm implementations do not match this fix.